src/test/bash/primedb.sh

```diff
@@ -2,11 +2,25 @@
 
 # Prime an empty DB with a few resources useful for testing with
 
-#if [[ $1 == "-h" || $1 == "--help" ]]; then
-#	echo "Usage: $0 [<name base>]"
-#	exit 0
-#fi
-#
+if [[ $1 == "-h" || $1 == "--help" ]]; then
+	cat << EOF
+
+Usage: $0
+
+Required Environment Variables:
+    EXCHANGE_ROOTPW: Exchange root password similar to what is set in exchange's config.json
+    EXCHANGE_USER: User in IBM org (set to anything to simply run tests)
+    EXCHANGE_PW: Password in IBM org (set to anything to simply run tests)
```

Comment:
Technically these aren't required because they have default values, so they should go in the Optional section.